### PR TITLE
isrcsubmit: init at 2.1.0

### PR DESCRIPTION
**Pull request by eduardosm**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/isrcsubmit/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "JonnyJD";
+    repo = "musicbrainz-isrcsubmit";
+    rev = "v${version}";
+    sha256 = "1lqs4jl2xv1zxmf0xsihk9rxzx2awq87g51vd7y3cq1vhj1icxqa";
+  };
+
+  propagatedBuildInputs = [ python3Packages.musicbrainzngs python3Packages.discid ];
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/isrcsubmit/default.nix:14]:
```suggestion
  propagatedBuildInputs = with python3Packages; [ musicbrainzngs discid ];
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 114971` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>isrcsubmit: <a href="https://termbin.com/7cqg">plain log</a>  |  <a href="https://htmlpreview.github.io/?https://gist.githubusercontent.com/SuperSandroBot/2e789f09a1eab3d7e8ae0dd90978df54/raw/index.html">pretty log </a></li>
  </ul>
</details>

File: pkgs/applications/audio/isrcsubmit/default.nix (hunk 1 above)
Comment (eduardosm) [reply pkgs/applications/audio/isrcsubmit/default.nix]: Fixed

**Comment by eduardosm** [comment]
> 1 package failed to build and already failed to build on hydra master:

It seems that drutil is required on Darwin. I don't think I can do much more than marking the package as broken for Darwin.

**Comment by SuperSandro2000** [comment]
> I don't think I can do much more than marking the package as broken for Darwin.

Fine for me.

**Comment by eduardosm** [comment]
Moved from applications/audio to tools/audio, which seems to be better suited.

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Pull request merged**